### DISCUSSION TITLE
Add support for custom parameters when logging in using providers

### DISCUSSION THREAD

**Discussion by valdeirpsr** [directus/directus#17968] [Feature Requests]
Google, for example, supports a few URL parameters during authentication: [**login_hint**](https://developers.google.com/identity/openid-connect/openid-connect?hl=pt-br#authenticationuriparameters)

It would be nice if Directus supported these optional values.

**Comment by valdeirpsr** [comment]
I did something similar to suppress a need

https://github.com/valdeirpsr/directus/commit/7db7582bd310a286866e6dbc290bb9ee3b657810

**Comment by connorwinston** [comment]
Hello @valdeirpsr! This does sound like a great feature. However, this feature request does not follow the Request-for-Comment (RFC) format we require to speed up the review and triaging process. Can you please update your discussion or create a new one using the correct format so it can be fully fleshed out?

_This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!_

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._

**Reply by valdeirpsr** [reply]
Thanks: https://github.com/directus/directus/discussions/18488
